### PR TITLE
chore(TransactionFeedV2): Add tracking for newly completed cross-chain swaps

### PR DESCRIPTION
**Pull request by sviderock**
### Description
9th PR for RET-1207. Implements tracking for newly completed cross-chain swaps.

### Test plan
Re-used existing corresponding test for the same functionality from existing feed.

### Related issues

- Relates to RET-1207

### Backwards compatibility
Yes

### Network scalability

If a new NetworkId and/or Network are added in the future, the changes in this PR will:

- [x] Continue to work without code changes, OR trigger a compilation error (guaranteeing we find it when a new network is added)

### CODE REVIEW

File: src/transactions/feed/TransactionFeedV2.tsx [hunk 1]
```diff
@@ -394,6 +451,29 @@ export default function TransactionFeedV2() {
...
+
+      const newlyCompletedCrossChainSwaps = standByTransactions.pending.filter(
+        (tx): tx is TokenExchange => completedCrossChainSwaps.includes(tx.transactionHash)
+      )
+
+      if (newlyCompletedCrossChainSwaps.length) {
+        trackCompletionOfCrossChainSwaps(newlyCompletedCrossChainSwaps)
+      }
+    },
+    [data?.transactions, standByTransactions.pending]
```
Comment (jeanregisser) [inline src/transactions/feed/TransactionFeedV2.tsx:474]: Is there a risk that the tracking could be triggered twice if one of these deps change without being in sync with the other?

**Review by jeanregisser — APPROVED** [review]
🚀

File: src/transactions/feed/TransactionFeedV2.tsx (hunk 1 above)
Comment (sviderock) [reply src/transactions/feed/TransactionFeedV2.tsx:474]: @jeanregisser You're right! Initially, I thought is shouldn't be a problem based on how the flow of `data?.transactions` and `standByTransactions` is tightly connected. But now as I think about it, it makes a lot more sense to utilize and add an extra field to `useNewlyCompletedTransactions` as it should be more precise in signaling the change.

**Comment by sviderock** [comment]
@jeanregisser updated

**Pull request merged**